src/Microsoft.ML/Runtime/EntryPoints/ImportTextData.cs

```diff
@@ -43,5 +53,16 @@ public static Output ImportText(IHostEnvironment env, Input input)
             var loader = host.CreateLoader(string.Format("Text{{{0}}}", input.CustomSchema), new FileHandleSource(input.InputFile));
             return new Output { Data = loader };
         }
+
+        [TlcModule.EntryPoint(Name = "Data.TextLoader", Desc = "Import a dataset from a text file", NoSeal = true)]
```

Comment:
>NoSeal = true [](start = 100, length = 13)

Just curious, what is the utility of having the class unsealed? Do we expect/want people to inherit from the auto-generated code?

Reply:
It is for convenience API for text loader. So that it can inherit all the loader settings but once we get rid of it then there would be no need for unsealing.

---
In reply to: [186547720](https://github.com/dotnet/machinelearning/pull/38#discussion_r186547720) [](ancestors = 186547720)